sea-orm-migration/tests/migrator/m20230109_000001_seed_cake_table.rs

```diff
@@ -0,0 +1,63 @@
+use sea_orm_migration::prelude::*;
+use sea_orm_migration::sea_orm::{entity::*, query::*};
+
+#[derive(DeriveMigrationName)]
+pub struct Migration;
+
+#[async_trait::async_trait]
+impl MigrationTrait for Migration {
+    async fn up(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        let db = manager.get_connection();
+
+        let transaction = db.begin().await?;
```

Comment:
Why open a transaction here? Just want to confirm, the behaviour should be no different whether we open a transaction or not? Because it's already inside a transaction (on Postgres)?

Reply:
1. I want to show it's possible to open a transaction inside migrate `up` & `down`.

2. For Postgres, yes, it's no difference. Because as you said you're already inside a transaction. However, it's not the case, for MySQL and SQLite. Opening a transaction is vital because the migration itself isn't atomic. If you want to seed the database atomically then you need to start a transaction yourself.

Reply:
Got it, cool